components/projects/EnvironmentVariableEditor.tsx

```diff
@@ -0,0 +1,192 @@
+import { ComponentProps, useCallback, useRef, useState } from "react";
+import { parseEnvFile, parseStringEnvContents } from "@/utils/helpers";
+import { EyeIcon, XIcon } from "lucide-react";
+import { useDropzone } from "react-dropzone";
+import { Button, Container, InputGroup } from "@/components/theme";
+import clsx from "clsx";
+
+export interface KeyPair {
+  envKey: string;
+  envValue: string;
+}
```

Comment:
I would change this to 👇🏼 so its self explanatory.
```suggestion
export interface EnvVariable {
  key: string;
  value: string;
}
```